src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * User interface for Duke application.
+ */
+public class Ui {
+    private Scanner sc;
+
+    /**
+     * Constructor for the user interface.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    public String getUserCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Returns boolean indicating whether there is further input from the user.
+     *
+     * @return true if there is further input, else false.
+     */
+    public boolean hasNextLine() {
+        return sc.hasNextLine();
+    }
+
+    /**
+     * Greets user with opening statements.
+     */
+    public void greet() {
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Says bye to user with closing statement.
+     */
+    public void sayBye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Tells user that file is not found.
+     */
+    public void fileNotFound() {
+        System.out.println("File not Found!");
+    }
+
+    /**
+     * Outputs list of tasks in Duke application.
+     *
+     * @param tasklist tasklist from Duke application.
+     */
+    public void list(TaskList tasklist) {
+        System.out.println("Here are the tasks in your list:");
+        for (int x = 0; x < tasklist.size(); x++) {
+            System.out.println(String.format("%s.%s", x + 1, tasklist.get(x).toString()));
+        }
```

Comment:
Just an observation, I think its convention to use `i` instead for a loop variable!